docs/getting-started/docker-playgrounds/flink_cluster_playground.md

```diff
@@ -0,0 +1,780 @@
+---
+title: "Flink Cluster Playground"
+nav-title: 'Flink Cluster Playground'
+nav-parent_id: docker-playgrounds
+nav-pos: 1
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+There are many ways to deploy and operate Apache Flink in various environments. Regardless of this
+variety, the fundamental building blocks of a Flink Cluster remain the same, and similar
+operational principles apply.
+
+In this playground you will see to how to run Jobs on Flink and get a good impression of how to 
+manage and operate them. More specifically you will:
+
+* Run a Flink Job
```

Comment:
For some reason I can't get this list and the TOC to render at the same time. It only shows up if comment out the table of contents. 

<img width="915" alt="Screen Shot 2019-07-31 at 4 16 03 PM" src="https://user-images.githubusercontent.com/1891970/62248832-ab3b9180-b3ae-11e9-88e8-1c98072105e1.png">

Reply:
Yep, the only way they will both render is if I add another sentence in between. This looks like a bug in Jekyll. 
<img width="1189" alt="Screen Shot 2019-07-31 at 4 20 25 PM" src="https://user-images.githubusercontent.com/1891970/62249078-3e74c700-b3af-11e9-83dd-fb180ff10814.png">

Reply:
I don't think its worth debugging Jekyll on this one, how about we just replace the bullet list with a sentence in the previous paragraph. 

> In this playground, you will learn how to manage and run Flink Jobs. You will see how to deploy and monitor an application, experience how Flink recovers from Job failure, and perform everyday operational tasks like upgrades and rescaling.

Reply:
Done. Thanks for spotting this!